charts/rstudio/README.md

```diff
@@ -0,0 +1,33 @@
+# RStudio Helm Chart
```

Comment:
I added a basic README, so we can ideally almost Copy & Paste from it when installing it for a user. Ideally we have a `README.md` in each chart directory.